test/rule-matches/nested-interactive-matches.js

```diff
@@ -0,0 +1,30 @@
+describe('nested-interactive-matches', function() {
+  var fixture = document.querySelector('#fixture');
+  var queryFixture = axe.testUtils.queryFixture;
+  var rule;
+
+  beforeEach(function() {
+    rule = axe._audit.rules.find(function(rule) {
+      return rule.id === 'nested-interactive';
+    });
+  });
+
+  afterEach(function() {
+    fixture.innerHTML = '';
+  });
+
+  it('should match if element has children presentational', function() {
```

Comment:
can you add a test that ensures this works for explicit roles too. Maybe a link with role=button?